registry/repository.go

```diff
@@ -65,6 +65,7 @@ type BlobStore interface {
 	content.Storage
 	content.Deleter
 	content.Resolver
+	TagFetcher
```

Comment:
Can we also make `Repository` implement `TagFetcher`, which defaults for manifests?